### ISSUE TITLE
dropout half of sdmap when testing cam_240_sd_model23.pt, result not good

### ISSUE THREAD

**Issue by zpc010** [jike5/P-MapNet#12]
![image](https://github.com/user-attachments/assets/9a0301e5-97d2-4bc2-963a-fbd835fb5d05)
![image](https://github.com/user-attachments/assets/a4797e28-2bac-4e7e-9186-19dea57e439d)

It seems like the model output depends mostly on sdmap, but not vision input. I also tried giving the model only sdmap as input, without vision input. The result is almost the same as it has both sdmap and vision input.

![image](https://github.com/user-attachments/assets/4d79451c-4bf7-4b01-a54e-42559e66e39d)

**Comment by zpc010** [comment]
Could you please provide the train log of that model cam_240_sd_model23.pt ？

**Comment by jike5** [comment]
> ![image](https://github.com/user-attachments/assets/9a0301e5-97d2-4bc2-963a-fbd835fb5d05) ![image](https://github.com/user-attachments/assets/a4797e28-2bac-4e7e-9186-19dea57e439d)
> 
> It seems like the model output depends mostly on sdmap, but not vision input. I also tried giving the model only sdmap as input, without vision input. The result is almost the same as it has both sdmap and vision input.
> 
> ![image](https://github.com/user-attachments/assets/4d79451c-4bf7-4b01-a54e-42559e66e39d)

The result looks interesting, It seems that the model overfits the sd map and should add some sd map dropout during training.

**Comment by jike5** [comment]
> Could you please provide the train log of that model cam_240_sd_model23.pt ？

I'm sorry. I tried looking for it, but I couldn't find it.
